modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -283,7 +303,9 @@ If you use PersistentVolumes, you can set the storage capacity for each volume.
 
 If Redpanda runs in a shared environment, where multiple applications run on the same worker node, you can make Redpanda less aggressive in CPU usage by enabling overprovisioning. This adjustment ensures a fairer distribution of CPU time among all processes, improving overall system efficiency at the cost of Redpanda's performance.
 
-You can enable overprovisioning by either setting the CPU request to a fractional value or setting `overprovisioned` to `true`.
+You can enable overprovisioning by either setting the CPU request to a fractional value or setting `resources.cpu.overprovisioned` to `true`.
+
+NOTE: When `resources.requests` or `resources.limits` are set, the `resources.cpu` parameter (including cores) is ignored. Ensure that you have not configured CPU requests and limits explicitly to avoid unexpected behavior in shared environments.
```

Comment:
This condition is that both requests and limits are set:
```suggestion
NOTE: When `resources.requests` and `resources.limits` are set, the `resources.cpu` parameter (including cores) is ignored. Ensure that you have not configured CPU requests and limits explicitly to avoid unexpected behavior in shared environments.
```

I'm not sure what the final sentence here means?